### DISCUSSION TITLE
Is RTK Query Inspector available in Redux DevTools?

### DISCUSSION THREAD

**Discussion by huyenltnguyen** [reduxjs/redux-toolkit#2615] [Q&A] [answered]
Hi, I'm wondering if RTK Query Inspector is already available in Redux DevTools.

According to the [Redux Essentials, Part 7: RTK Query Basics doc](https://redux.js.org/tutorials/essentials/part-7-rtk-query-basics#query-arguments-and-cache-keys), RTK Query support is still a WIP, but [the linked PR](https://github.com/reduxjs/redux-devtools/pull/750) as well as [its tracking issue](https://github.com/reduxjs/redux-toolkit/issues/1097) are closed, and Mark commented in the issue that RTKQ Inspector is available in Redux DevTools v3.

I'm using Redux DevTool v3.0.11, which I believe is the latest version, but I'm not seeing RTKQ Inspector, so I just wanted to check if this is because it is not available or I'm having issues with my Redux config.

**Comment by markerikson** [comment, marked as answer]
Yes, it was released as part of Redux Devtools v3.  I need to update that callout box.

There should be an "RTK Query" tab at the bottom of the Redux DevTools UI, next to "Chart":
![image](https://user-images.githubusercontent.com/1128784/185479071-1bdf5b33-d1c9-4488-906c-5c868c2f10de.png)

**Reply by huyenltnguyen** [reply]
Ah hah! I was having the browser console docked on the side so I couldn't t see it

<img width="541" alt="Screen Shot 2022-08-19 at 09 14 59" src="https://user-images.githubusercontent.com/25715018/185528161-145de088-1b2a-485b-874d-baafa17db61a.png"> 

Thank you @markerikson!

Also, thanks @FaberVitale for the awesome work!
